### ISSUE TITLE
wm and wa macros has inconsistend output in case of pseudo motors

### ISSUE THREAD

**Issue by reszelaz** [sardana-org/sardana#929]
Reported by @jordiandreu 

This is the output from the spock door, which is kind of bizarre:

```
door_sats [18]: wm E
E
User 
High 30.0
Current 6.50000456987
Low 3.5
Dial 
High 30.0
Current 6.50000456987
Low 3.5

door_sats [19]: wa E
Current positions (user, dial) on 2016-11-15 13:24:17.638568

E
User 6.5000
Dial nan
```

Simply, one has dial position the other does not.

**Comment by reszelaz** [comment]
This was fixed in #953. Now pseudo motors show position as dial position. Hope this is what you expected @jordiandreu. If not please re-open.

**Comment by reszelaz** [comment]
On the other hand it may be confusing to see a DialPosition attribute value for a pseudo motor knowing that this attribute exists only for physical motors. Anyone has a similar feeling?
